src/microbe_stage/editor/upgrades/OrganelleUpgradeGUI.cs

```diff
@@ -112,10 +113,12 @@ public override void _Ready()
 
                 var selectionButton = upgradeSelectionButtonScene.Instance<MicrobePartSelection>();
 
+                var cost = (int)Math.Min(upgrade.MPCost * costMultiplier, 100);
```

Comment:
Is this the same math when calculating the cost of the action when it is performed? If this isn't then I worry that this might be rounded differently and give a wrong reading in the GUI...

Reply:
Sorry, totally forgot that I made this PR 🤦‍♂️

Now that I think about it, perhaps getting the cost with `Editor.WhatWouldActionsCost` would be better as it account the history?

Reply:
That would actually make a lot of sense, and would also solve the situation where some upgrades are refunded at the same time as swapping to something else.

Reply:
Alright, I finally opened a PR for it https://github.com/Revolutionary-Games/Thrive/pull/4284.